packages/strapi-connector-bookshelf/lib/formatter.js

```diff
@@ -38,7 +38,7 @@ const defaultFormatter = {
     }
   },
   date: value => {
-    const cast = new Date(value);
+    const cast = parseISO(value);
     return isValid(cast) ? formatISO(cast, { representation: 'date' }) : null;
```

Comment:
We can extract only the date part from the UTC ISO so it stays consistent all the way. we could consider doing this

```suggestion
    return isValid(cast) ? cast.toISOString().split('T')[0] : null
```